Lib/_pylong.py

```diff
@@ -115,7 +167,9 @@ def inner(n, w):
     # only if the number has way more than 10**15 digits, that exceeds
     # the 52-bit physical address limit in both Intel64 and AMD64.
     w = int(w * 0.3010299956639812 + 1)  # log10(2)
-    pow10_cache = {}
+    pow10 = compute_powers(w, 5, 1000)
```

Comment:
Now we may want to introduce a named constant for `1000` which is used twice.

Reply:
Good catch! Done.